docs/billing-integration.md

```diff
@@ -82,6 +82,9 @@ data:
     # Metric names should match verbatim the identifiers created
     # during pricing setup.
     metrics:
+    # The app must send usage reports to the agent for this metric
+    # since it is not an instance time metric (which the agent can
```

Comment:
I think it's more precise to say that metrics are either generated by "native" sources or externally reported. Since this metric is not declared in the sources section, it must be externally reported

Reply:
Updated to:
```
    # Metric usage reports are either generated by the agent as
    # defined in "sources", or externally, i.e. by the app.
    # Since this metric is not supported by the "sources" section,
    # the app must send usage reports for this metric to the agent.
```